app/controllers/users_controller.rb

```diff
@@ -13,6 +13,63 @@ def show
     end
   end
 
+  def login_form
```

Comment:
do we actually need login_form or login methods anymore after we have the create method that uses OmniAuth?